injected/package.json

```diff
@@ -3,31 +3,22 @@
   "scripts": {
     "postinstall": "npm run copy-sjcl",
     "copy-sjcl": "node scripts/generateSJCL.js",
+    "build": "npm run build-types && npm run build-locales && npm run bundle-trackers && npm run bundle-entry-points",
     "bundle-config": "node scripts/bundleConfig.mjs",
-    "build": "npm run build-types && npm run build-locales && npm run bundle-trackers && npm run build-firefox && npm run build-chrome && npm run build-apple && npm run build-android && npm run build-windows && npm run build-integration && npm run build-chrome-mv3",
+    "bundle-entry-points": "node scripts/entry-points.js",
+    "build-chrome-mv3": "node scripts/entry-points.js",
     "build-locales": "node scripts/buildLocales.js",
-    "build-firefox": "node scripts/entry-points.js --platform firefox",
-    "build-chrome": "node scripts/entry-points.js --platform chrome",
-    "build-chrome-mv3": "node scripts/entry-points.js --platform chrome-mv3",
-    "build-apple": "node scripts/entry-points.js --platform apple && node scripts/entry-points.js --platform apple-isolated",
-    "build-android": "node scripts/entry-points.js --platform android && node scripts/entry-points.js --platform android-autofill-password-import && node scripts/entry-points.js --platform android-broker-protection",
-    "build-windows": "node scripts/entry-points.js --platform windows",
-    "build-integration": "node scripts/entry-points.js --platform integration",
     "build-types": "node scripts/types.mjs",
     "bundle-trackers": "node scripts/bundleTrackers.mjs --output ../build/tracker-lookup.json",
     "test-unit": "jasmine --config=unit-test/config.json",
-    "test-int": "npm run build-integration && npm run playwright",
+    "test-int": "npm run playwright",
     "test-int-x": "xvfb-run --server-args='-screen 0 1024x768x24' npm run test-int",
     "test": "npm run lint && npm run test-unit && npm run test-int && npm run playwright",
     "serve": "http-server -c-1 --port 3220 integration-test/test-pages",
     "playwright": "playwright test --grep-invert '@screenshots'",
```

Comment:
Does "playwright" no longer build?

Reply:
it does it on demand now, in the playwright file. This is preferred to ensure a fresh build when playwright is run from the command line or IDE/editor (which we wouldn't have done before because of the speed)

Reply:
Ah yeah I see, cheers